src/Assets/PaymentMethodAssets.php

```diff
@@ -30,24 +30,15 @@ class PaymentMethodAssets {
 	 */
 	private $asset_registry;
 
-	/**
-	 * An instance of the Asset Api
-	 *
-	 * @var Api
-	 */
-	private $asset_api;
-
 	/**
 	 * Constructor for class
 	 *
 	 * @param PaymentMethodRegistry $payment_method_registry An instance of Payment Method Registry.
 	 * @param AssetDataRegistry     $asset_registry  Used for registering data to pass along to the request.
-	 * @param Api                   $asset_api       Used for registering scripts and styles.
 	 */
-	public function __construct( PaymentMethodRegistry $payment_method_registry, AssetDataRegistry $asset_registry, Api $asset_api ) {
+	public function __construct( PaymentMethodRegistry $payment_method_registry, AssetDataRegistry $asset_registry ) {
```

Comment:
Removes the Assets/Api class because this is something that is coupled to the plugin path for blocks. So it's interfaces will not work for extensions registering payment method types.